### ISSUE TITLE
"Edit" in chat shows entities as encoded, then re-encodes them

### ISSUE THREAD

**Issue by kingjon3377** [PenoaksDevOps/HolyWorlds-Issues#31]
As a Moderator (or perhaps a mere Editor), in Chat, if I Edit a post containing a special character (ampersand or double-quote, and probably others), the dialog that pops up to allow me to edit it will show me that character as its HTML-encoded form (e.g. `&amp;`). 
![edit_1](https://user-images.githubusercontent.com/1522278/102147921-8fb51780-3e39-11eb-9b7f-051d5dbf3d20.png)

If I click to save without changing the entities to what they represent, the edited chat message will appear as it did in the dialog (except for BBCode being interpreted a second or so later):
![edit_2](https://user-images.githubusercontent.com/1522278/102148164-f20e1800-3e39-11eb-87f9-6d33059ba6a5.png)

If one isn't paying attention, this can of course snowball with repeated edits:
![edit_3](https://user-images.githubusercontent.com/1522278/102148247-1f5ac600-3e3a-11eb-84b2-525390ba50ab.png)

But since an alert editor will probably notice this immediately (unless the chat room suddenly returned to the level of activity we saw a decade ago) I would say fixing this should probably not be a particularly high-priority item at this point; I merely wanted to make sure it was noted somewhere other than my own memory.

**Comment by aidan-casey** [comment]
Yeah, this is a known issue. We are intending to overhaul the chat room, so I think we just keep an eye on this for now and resolve at that point. 👍🏼

**Comment by BarelyAPrincess** [comment]
To be honest I didn’t even know the chat was still working to any degree. The intention was for the HolyWorlds discord server to be a temporary alternative to chat until we could get around to overhauling the chat.

**Comment by aidan-casey** [comment]
Chat is in "mostly working" order. The avatar API script broke on the first server transition when Penoaks took over HW and the historical log with it. Both due to missing dependencies (I believe jQuery UI for the historical log).

An Ajax chat is not a long term solution, but that is where we are at.

This specific issue would be an easy fix, but I'm just not sure it's worth the time when our ultimate goal is a websocket solution.
